packages/wallet-sdk/src/sign/scw/SCWSigner.ts

```diff
@@ -78,30 +84,99 @@ export class SCWSigner implements Signer {
     this.handleResponse(args, decrypted);
   }
 
-  async request(request: RequestArguments) {
+  async initSubAccountConfig() {
```

Comment:
Does this need to be a method on the class. I wonder if we should separate it out into a small utility and then write some simple tests for it